ci_scripts/ci_post_clone.sh

```diff
@@ -39,9 +38,12 @@ EOF
   export QT_MACOS_BIN=`pwd`/qt_static_macos/qt6/bin
   export PATH=`pwd`/qt_static_macos/qt6/bin:$PATH
 else
-  pip3 install aqtinstall
-  aqt install-qt -O /Volumes/workspace/repository/qt_ios mac desktop $QTVERSION -m qtwebsockets qt5compat
-  aqt install-qt -O /Volumes/workspace/repository/qt_ios mac ios $QTVERSION -m qtwebsockets qt5compat
+  curl -L https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/mozillavpn.v2.mozillavpn.cache.level-1.toolchains.v3.qt-ios.latest/artifacts/public%2Fbuild%2Fqt6_ios.zip --output qt_ios.zip
+  unzip qt_ios.zip
```

Comment:
Note: before landing this need to change from `cache.level-1` to `cache.level-3` - so we only use changes from /main or a release branch, not pr's